marine-examples/multiservice_marine_test/README.md

```diff
@@ -0,0 +1,117 @@
+# Multiservice Marine Test
+
+## Overview
+
+This example illustrates another ability of the `marine_test` macro: testing several services at once. To show that we will create couple of services: a producer service which will create some data, and a consumer service which will process the data. Then test it ensuring that the consumer properly processed data from the producer.
+
+## Build
+
+To build the example, please run the following command:
+
+```
+./build.sh
+```
+
+Upon a successful build it results a couple of wasm modules of the producer and consumer in the respective directories:
+
+```
+➜  multiservice_marine_test git:(main) ✗ ls producer/artifacts
+producer.wasm
+➜  multiservice_marine_test git:(main) ✗ ls consumer/artifacts
+consumer.wasm
+```
+
+## Run
+
+One of the ways to run the tests is:
+
+> Please make sure you're using a nightly toolchain using `rustup show`.
+> If not, please do that with the `rustup override set nightly` for the current directory.
+
+```
+cargo test
+```
+
+Upon successful run it results the following:
+
+```
+   Finished test [unoptimized + debuginfo] target(s) in 3m 05s
+     Running unittests src/main.rs (target/debug/deps/multiservice_marine_test-20dd18fbb0e3ed6d)
+
+running 2 tests
+test tests_on_mod::test ... ok
+test tests::test ... ok
+
+test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 32.87s
+
+```
+
+We can also use the `mrepl` tool to test the service.
+
+To use a producer, please run `mrepl` in the `producer` directory:
+
+```
+➜  multiservice_marine_test git:(main) ✗ cd producer
+➜  producer git:(main) ✗ mrepl Config.toml
+Welcome to the Marine REPL (version 0.16.1)
+Minimal supported versions
+  sdk: 0.6.0
+  interface-types: 0.20.0
+
+New version is available! 0.16.1 -> 0.16.2
+To update run: cargo +nightly install mrepl
```

Comment:
Maybe update it and remove hte lines about update from doc? They don't look relevant